documentation/onboard-dpg-in-sdkautomation/README.md

```diff
@@ -0,0 +1,41 @@
+# Service Onboard DPG with Swagger CI Pipeline
+
+Service Onboarding DPG with Swagger CI pipeline can help you get generated SDK and ApiView during the procedure of creating swagger PR.
+
+This Document describes the process of how to onboard DPG with SDK automation pipeline. __This document is targeting for service team.__ 
+
+*If you want to go through details of pipeline implementation, please go to [Integrate DPG and ApiView in SDK Automation Pipeline](Integrate-dpg-and-apiview-in-sdk-automation-pipeline.md).*
+
+## WorkFlow
+
+![workflow-for-service-team](workflow-service-team.png)
+
+__Description:__
+1. Creates a swagger PR.
+2. If the service has onboarded DPG before, please go to step 4. Otherwise, please go to step 3.
```

Comment:
```suggestion
2. If the SDK was generated for this service bfore , please go to step 4. Otherwise, this is a new service and please go to step 3.
```